Source/WebCore/css/calc/CSSCalcTree+Parser.cpp

```diff
@@ -702,6 +703,20 @@ static std::optional<TypedChild> consumeProgress(CSSParserTokenRange& tokens, in
     return TypedChild { makeChild(WTFMove(op), *outputType), *outputType };
 }
 
+static std::optional<TypedChild> consumeLengthPercentage(CSSParserTokenRange& tokens, int depth, ParserState& state)
```

Comment:
The goal is `consumeAnchor` will use this function to parse the fallback value too (in a separate patch)

Reply:
Now it's using `consumeValueWithoutSimplifyingCalc`